Sensus.Android/Sensus.Android.csproj

```diff
@@ -36,11 +36,6 @@
     <AndroidLinkMode>None</AndroidLinkMode>
     <AndroidSupportedAbis>armeabi;armeabi-v7a;x86</AndroidSupportedAbis>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <CustomCommands>
-      <CustomCommands>
-        <Command type="BeforeBuild" command="touch Assets/UnitTestingProtocol.json" workingdir="${ProjectDir}" externalConsole="True" />
-      </CustomCommands>
-    </CustomCommands>
```

Comment:
Looks like this is still deleted within your PR. Easiest thing to do is close Xamarin Studio, open the project file in a text editor, and copy the above back into the file at the correct location.